From e6070ee923681613a4e17600678ec38a967b1d88 Mon Sep 17 00:00:00 2001
From: Samuel Verschelde <stormi-xcp@ylix.fr>
Date: Thu, 20 May 2021 16:09:13 +0200
Subject: [PATCH 13/13] Fix FreePBX detection when centos-release is missing

For some reason, some FreePBX systems have a centos-release file and
others don't. So we read sangoma-release first, which should always be
present (and if it isn't we still check centos-release just in case)

Signed-off-by: Samuel Verschelde <stormi-xcp@ylix.fr>
---
 mk/xe-linux-distribution | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mk/xe-linux-distribution b/mk/xe-linux-distribution
index 63cef13..b5c174e 100755
--- a/mk/xe-linux-distribution
+++ b/mk/xe-linux-distribution
@@ -586,6 +586,7 @@ if [ -z "${TEST}" ] ; then
     identify_lsb lsb_release         && exit 0
     identify_debian /etc/debian_version && exit 0
     identify_boot2docker /etc/boot2docker && exit 0
+    identify_sangoma /etc/sangoma-release && exit 0
     identify_sangoma /etc/centos-release && exit 0
     identify_freebsd && exit 0
 
-- 
2.30.2

